### ISSUE TITLE
Allow for re-curation

### ISSUE THREAD

**Issue by wbazant** [ebi-gene-expression-group/atlas-ingest#1]
Hello! I was being nosy and checking out what https://github.com/ebi-gene-expression-group is up to. Congratulations on taking what seems to be a really good approach on metadata curation.

I've worked on Atlas until 2017 and later made a much humbler website with functional genomic data, as a WormBase ParaSite component and based on ISL data, and tried to learn from a few things I saw in Atlas. 

One thing I've learned was that curated experiments will differ and that will mess up any aggregate displays over time.

For example, for S. mansoni we have these studies:
http://test.parasite.wormbase.org/expression/schistosoma_mansoni_prjea36577/

The metadata is aggegated in this display (click "select tracks" on the left"):
http://parasite.wormbase.org/jbrowse/index.html?data=%2Fjbrowse-data%2Fschistosoma_mansoni_prjea36577%2Fdata&loc=SM_V7_1%3A24486826..24488380&tracks=DNA%2CGene_Models&highlight=
in which I can see that I've used "Cercariae" in one study, "cercariae" in the other, and now they don't align. So it needs to be fixed.

I keep all curation in a Git repository:
https://github.com/WormBase/wbps-expression/tree/master/studies/schistosoma_mansoni

so fixing the metadata is as easy as editing files, and then rerunning some stuff.

I recommend you allow for re-doing experiments in your model. It's gonna happen: the curator says they're finished, they load the experiment to wwwdev, and something looks wrong. So have a plan for how to handle this case.

**Comment by wbazant** [comment]
(feel free to close the issue whenever convenient)

**Comment by hewgreen** [comment]
Hi Wojtek,

Thanks for the nosiness. Nice to know someone is checking out what we are doing. This repo will eventually house some curator/wrangler tooling that can be triggered in many ways from our project builder automatically and manually. The code is young at the moment and will work alongside old systems while it's being tested.

Editing the metadata files in the system (aka recuration) will retrigger validation and ingestion and may or may not trigger reanalysis depending on the change made. The analysis pipelines are smart enough to work that out. We are working on improving tracking provenance of curation activity by introducing versioning and revamping status tracking. The initial phase of this code will be to implement a status tracker to help with this.

Please keep checking in

**Comment by wbazant** [comment]
@hewgreen

Tracking changes to curation is probably a good idea, I've always been a fan. I remember I've set up a fg_atlas cron job in something like 2017: do an rsync with a pattern to an offline repository, and make a commit. After a few months we've had a broken experiment after an accidental change and I was able to figure it out and restore the previous version. It may still be there, forgotten, unwanted- but still, archiving.

Actually, that's the only time I'm aware of that this scheme to archive experiment files was useful, apart from this potential new use case where you bulk update metadata across the portal to make equivalent assay groups presented together on the gene page.
